PresentScreenings/Utilities/ControlsFactory.cs

```diff
@@ -65,6 +67,13 @@ public static NSButton NewCancelButton(CGRect frame)
             return cancelButton;
         }
 
+        public static NSButton NewCheckbox(CGRect frame)
+        {
+            var box = NewStandardButton(frame);
+            box.SetButtonType(NSButtonType.Switch);
+            return box;
+        }
```

Comment:
Good addition!
Even though there is not much to it now, it will help ensure that these controls will look and feel the same across the application in the future.

Reply:
Thanks :-)